operations/workplace/people/hr-cadences.md

```diff
@@ -10,15 +10,74 @@ Listening sessions are 1x1 meeting with your HR partner. The intent of the HR Li
+
+### Cadence
+* Every 6 months, March and September 
+* [additional logistics can be found here](https://handbook.mattermost.com/operations/operations/company-cadence#staff-enablement-survey)
+* Post-action planning at team and company level 
+
+### Past Surveys 
+* [March 2020 - Staff Engagement Survey](https://docs.google.com/presentation/d/1bkUQOCvJyYX-695gpZI7SJal14AWefFx5fJQxTOJKlI/edit#slide=id.g730cbfe72a_0_9)
+
+For additional information on Gallup [see the Gallup website](https://www.gallup.com/corporate/212381/who-we-are.aspx). You can review the list of questions ahead of taking the survey. Each question is important to Mattermost as it relates to our leadership principles, remote teams and shared values of Trust, Growth and Iteration.  
+
+#### **Q1: I know what is expected of me at Mattermost.** 
+ 
+Role clarity and alignment is vital at any company. This is especially important for our remote teams and is a significant reason Mattermost uses a variety of methods to align expectations with individuals, within teams and across teams. This includes performance reviews to discuss expectations and give feedback, [AORs](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#aor) to help cross-team collaboration, project planning with clear [DRIs](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#dri) that create transparency on roles within project teams and QBRs that provide team and individual focus as it relates to Mattermost’s Mission and Vision. Role clarity requires a continuous dialogue with one’s Manager. In order to [earn](https://handbook.mattermost.com/company/about-mattermost#leadership-principles) and keep [trust](https://docs.google.com/presentation/d/1BDSaeW-M92gth_NM1vI23dtbFLBsYVIk/edit#slide=id.g6f3b66b1fb_0_352), conversations around expectations are important to avoid misalignment of expectations which can cause frustration, productivity and performance issues. 
+
+#### **Q2: I have the materials and equipment I need to do my work right.** 
+Mattermost encourages staff to [customize their home workspace](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-spend-company-money#equipment) to best fit individual needs. Beyond equipment and home office materials can include any resource that would help optimize productivity and minimize distractions. Tools can include things like: systems, hardware or internal documentation. There may not be a budget or resources available to approve every tool request but alternative or interim solutions should be a collaborative effort to [grow](https://docs.google.com/presentation/d/1BDSaeW-M92gth_NM1vI23dtbFLBsYVIk/edit#slide=id.g6f3b66b1fb_0_352) our individual and team capabilities.   
```

Comment:
```suggestion
Mattermost encourages staff to [customize their home workspace](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-spend-company-money#equipment) to best fit individual needs. Beyond equipment and home office this can include any resource that would help optimize productivity and minimize distractions. Tools can include things like: Systems, hardware, or internal documentation. There may not be a budget or resources available to approve every tool request but alternative or interim solutions should be a collaborative effort to [grow](https://docs.google.com/presentation/d/1BDSaeW-M92gth_NM1vI23dtbFLBsYVIk/edit#slide=id.g6f3b66b1fb_0_352) our individual and team capabilities.   
```